### PR TITLE
feat: (DHSC-55) drupal security update - 10.2.9

### PR DESCRIPTION
**Pull request by cedrichillion-manifesto**

### CODE REVIEW

File: composer.json
Comment (jackholding) [inline composer.json:1]: @cedrichillion-manifesto I might've expected `drupal/core-dev` package to also be updated as part of this. I find it best to use the command listed here when upgrading the patch version: [Update to the latest patch version of core](https://www.drupal.org/docs/updating-drupal/updating-drupal-core-via-composer#s-update-to-the-latest-patch-version-of-core)

**Review by jackholding — CHANGES_REQUESTED** [review]
@cedrichillion-manifesto I've left a couple of things to look at, thanks

File: composer.json
Comment (cedrichillion-manifesto) [reply composer.json:1]: @jackholding , like so?

Comment (jackholding) [reply composer.json:1]: Awesome! That's resolved the pipeline 👍  How did you update only the facets module in the end then?

**Review by jackholding — APPROVED** [review]

**Pull request merged**